moveit2.repos

```diff
@@ -47,3 +47,10 @@ repositories:
     # version: ros2
     url: https://github.com/PickNikRobotics/octomap_msgs
     version: ros2
+
+# Temporarily install OMPL from melodic
+# TODO(henningkayser): remove once OMPL is released for ROS2
```

Comment:
Love this note. Does @mamoll know about this todo? He releases OMPL

Reply:
He will, in the release announcement ;)